test/TEST-04-JOURNAL/test.sh

```diff
@@ -7,4 +7,10 @@ TEST_DESCRIPTION="Journal-related tests"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_append_files() {
+    (
```

Comment:
Why `()` is necessary here?

Reply:
I'm not so familiar with systemd test framework but all `test_append_files` uses subshell to run commands. So did I.

Reply:
OK. Thanks.